R/commit.R

```diff
@@ -1,22 +1,26 @@
 #' Stage and commit changes
 #'
-#' @description To commit changes, start with *staging* the files to be included
-#' in the commit using [git_add()] or [git_rm()]. Use [git_status()] to see an
-#' overview of staged and unstaged changes, and finally [git_commit()] creates
-#' a new commit with currently staged files.
+#' @description
+#' To commit changes, start by *staging* the files to be included in the commit
+#' using `git_add()` or `git_rm()`. Use `git_status()` to see an overview of
+#' staged and unstaged changes, and finally `git_commit()` creates a new commit
+#' with currently staged files.
 #'
-#' [git_commit_all] is a shorthand that will automatically stage all new and
```

Comment:
I've removed the misleading "automatically stage all new and modified" and been explicit about how new files are (not) handled.